deploy/README.md

```diff
@@ -39,3 +39,30 @@ In `frontend/app/.env.local`:
 docker-compose build
 docker-compose up -d
 ```
+
+OR using makefile
+
+```bash
+make build-prod
+make start-prod
+make log-prod
+make stop-prod
+```
+
+## Test
+
+For those who want to test before
```

Comment:
Le mot "test" est communément employé pour les tests automatisés. Pour éviter de la confusion, ce serait mieux d'utiliser "dev" (notamment pour le nom du fichier docker-compose) et peut-être changer la phrase ci-dessus en "For those who want to try this Docker configuration in a local environment"

Reply:
OK, je changerai ça, merci.